packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -108,6 +108,7 @@ declare global {
       namespace: AutomationElementId
       hideCommandLog: boolean
     }
+    __Cypress__: boolean
```

Comment:
Not related to this PR but it feels weird that our global Window definitions are in a `graphql` specific file. `__Cypress__` is derived from the `index.html` file.

Reply:
Yeah it's weird. I wouldn't object to moving this to anywhere that's more universal, I'd like to keep it all together though.